components/styled/icon/src/atoms/info-circle-icon.tsx

```diff
@@ -6,11 +6,11 @@ import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
 import { ICON_COLOURS, calculateIconSize, IconStatus, IconSize } from '@ltht-react/styles'
 
 const calculateIconColor = (status: string): SerializedStyles => css`
-  ${status === 'green' && `${ICON_COLOURS.SUCCESS.VALUE};`}
-  ${status === 'amber' && `${ICON_COLOURS.WARNING};`}
-  ${status === 'red' && `${ICON_COLOURS.DANGER};`}
-  ${status === 'default' && `${ICON_COLOURS.DEFAULT};`}
-  ${status === 'info' && `${ICON_COLOURS.INFO};`}
+  ${status === 'green' && `${ICON_COLOURS.SUCCESS.VALUE} !important;`}
```

Comment:
Is there a nicer way to achieve this without using `!important` ? From experience this usually causes more issues than it solves